package.json

```diff
@@ -11,6 +11,7 @@
   },
   "scripts": {
     "build": "webpack --mode production",
+    "dev": "webpack --mode development --watch",
```

Comment:
command to watch the changes and auto-build in dev mode.
Super useful for debugging the package.

Reply:
nice!